CMakeLists.txt

```diff
@@ -63,6 +66,13 @@ target_compile_features(${POCLIB}
     cxx_std_14
 )
 
+# TESTING
+# ----------------------------------------
+if(BUILD_TESTING)
+  message(STATUS "Configuring tests")
+  add_subdirectory(test)
+endif()
```

Comment:
I would suggest we don't use `BUILD_TESTING` as this tends to turn on *all* the testing of all dependencies. If ports is built in-tree, this could be annoying. I would prefer a `TEST_PORTS_OF_CALL` or `PORTS_BUILD_TESTING` or something similar.